lib/Frontend/nvqpp/ConvertExpr.cpp

```diff
@@ -1592,6 +1592,13 @@ bool QuakeBridgeVisitor::VisitCallExpr(clang::CallExpr *x) {
                                          reportNegateError, isAdjoint,
                                          /*control=*/true);
 
+    if (funcName.equals("u3")) {
+      const auto size = args.size();
+      assert(size == 4);
```

Comment:
Can you add a comment here? Something like `&& "The u3 operator must have 4 arguments"` would be great.

Reply:
Also should we not be leveraging `isControl` and/or `negations` and/or `isAdjoint`?